otter/test/convergence/test_steps.py

```diff
@@ -469,9 +469,9 @@ def test_add_nodes_to_clb_terminal_failures(self):
         if there is any other 4xx error, then
         the error is propagated up and the result is a failure.
         """
-        terminals = (CLBDeletedError(lb_id=u"12345"),
+        terminals = (CLBNotFoundError(lb_id=u"12345"),
+                     CLBDeletedError(lb_id=u"12345"),
                      CLBNodeLimitError(lb_id=u"12345"),
-                     NoSuchCLBError(lb_id=u"12345"),
```

Comment:
I think it's important to pass in an instance of CLBNotFoundError too since it proves that we handle all of them equally. And I left the CLBDeletedError there to prove that subclasses are honored